lib/steps/fetch_step.go

```diff
@@ -6,6 +6,8 @@ import (
 	"github.com/Originate/git-town/lib/script"
 )
 
+var fetched bool
```

Comment:
I don't think this is needed as I updated fetch to always run in preconditions. I'll remove this